octobot_trading/modes/channel/abstract_mode_consumer.py

```diff
@@ -69,17 +69,24 @@ async def internal_callback(self, trading_mode_name, cryptocurrency, symbol, tim
                              f"the order being refused by the exchange.")
 
     def get_minimal_funds_error(self, symbol, final_note):
-        market_status = self.exchange_manager.exchange.get_market_status(symbol, price_example=None, with_fixer=False)
-        try:
-            base, quote = symbol_util.parse_symbol(symbol).base_and_quote()
-            portfolio = self.exchange_manager.exchange_personal_data.portfolio_manager.portfolio
-            funds = {
-                base: portfolio.get_currency_portfolio(base),
-                quote: portfolio.get_currency_portfolio(quote)
-            }
-        except Exception as err:
-            self.logger.error(f"Error when getting funds for {symbol}: {err}")
-            funds = {}
+        if symbol is None:
+            return (
+                f"Not enough funds to create new orders after {final_note} evaluation: "
+                f"{self.exchange_manager.exchange_name} exchange minimal order "
+                f"volume has not been reached."
+            )
```

Comment:
That's the reason why symbol is None?

Reply:
in index trading, we don't send a given symbol to buy, we tell the consumer to "rebalance the index" and the consumer will create orders accordingly (it's not symbol related).

Currently:
- producer decides if a rebalance should happen
- consumer executes the rebalance

Reply:
>  (it's not symbol related).

That's my point, I don't understand why we trigger this error here when symbol is None

Reply:
This function's goal is to log an error, if it's called it's that the error already occured and we want inform the user that an error occured. This symbol condition allows to create the error without given symbols, which is a case that can now happen